src/Core/ServiceWrapper/winsw.csproj

```diff
@@ -14,10 +14,12 @@
     <Copyright>Copyright 2008-2016 Oleg Nenashev, CloudBees, Inc. and other contributors</Copyright>
     <RootNamespace>winsw</RootNamespace>
     <AssemblyName>WindowsService</AssemblyName>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(TargetFramework)' != 'netcoreapp3.1'">
-    <ILMergeVersion>3.0.40</ILMergeVersion>
+    <SignAssembly>false</SignAssembly>
```

Comment:
All changes to *.csproj* files should actually be reverted.

Reply:
OK